### ISSUE TITLE
Use system-wide utf8proc

### ISSUE THREAD

**Issue by tieugene** [corporateshark/WCMCommander#503]
Some linux distros (like Fedora/RHEL) disallow to bundle libraries if they exist as separate pakages.

**Comment by tieugene** [comment]
Dirty hack:
https://tieugene.fedorapeople.org/rpms/wcm/FindUtf8proc.cmake
https://tieugene.fedorapeople.org/rpms/wcm/wcm-utf8proc.patch

**Comment by corporateshark** [comment]
This patch breaks OS X.

**Comment by corporateshark** [comment]
Btw, utf8proc is used only on OS X.

**Comment by tieugene** [comment]
This is Linux-specific dirty patch. As an example.

**Comment by corporateshark** [comment]
It is not used on Linux anyway. Apple-only.

**Comment by tieugene** [comment]
Ok, then it must be removed in cmake script for Linux/Windows.
Right?

**Comment by corporateshark** [comment]
Yep. And some defines inside the code should be fixed using '''__APPLE__'''. I don't have a Linux machine here to test it.

**Comment by tieugene** [comment]
I have. Let's try, I'll help.
